install_config/web_console_customization.adoc

```diff
@@ -19,6 +19,54 @@ using extensions, which let you run scripts and load custom stylesheets when the
 web console loads. You can change the look and feel of nearly any aspect of the
 user interface in this way.
 
+== Configuring Web Console Customizations with Ansible
+
+During 
+xref:../install_config/install/advanced_install.adoc#install-config-install-advanced-install[advanced installations], 
+many modifications to the web console can be configured using 
+xref:../install_config/install/advanced_install.html#advanced-install-configuring-global-proxy[the following parameters], which are configurable in the inventory file:
+
+- xref:changing-the-logout-url[`*openshift_master_logout_url*`]
+- xref:loading-custom-scripts-and-stylesheets[`*openshift_master_extension_scripts*`]
+- xref:loading-custom-scripts-and-stylesheets[`*openshift_master_extension_stylesheets*`]
+- xref:serving-static-files[`*openshift_master_extensions*`]
+- xref:serving-static-files[`*openshift_master_oauth_template*`]
+- xref:../install_config/cluster_metrics.adoc#install-config-cluster-metrics[`*openshift_master_metrics_public_url*`]
+- xref:../install_config/aggregate_logging.adoc#install-config-aggregate-logging[`*openshift_master_logging_public_url*`]
+
+.Example Web Console Customization with Ansible
+====
+----
+# Configure logoutURL in the master config for console customization
+# See: https://docs.openshift.com/enterprise/latest/install_config/web_console_customization.html#changing-the-logout-url
+#openshift_master_logout_url=http://example.com
+
+# Configure extensionScripts in the master config for console customization
+# See: https://docs.openshift.com/enterprise/latest/install_config/web_console_customization.html#loading-custom-scripts-and-stylesheets
+#openshift_master_extension_scripts=['/path/to/script1.js','/path/to/script2.js']
+
+# Configure extensionStylesheets in the master config for console customization
+# See: https://docs.openshift.com/enterprise/latest/install_config/web_console_customization.html#loading-custom-scripts-and-stylesheets
+#openshift_master_extension_stylesheets=['/path/to/stylesheet1.css','/path/to/stylesheet2.css']
+
+# Configure extensions in the master config for console customization
+# See: https://docs.openshift.com/enterprise/latest/install_config/web_console_customization.html#serving-static-files
+#openshift_master_extensions=[{'name': 'images', 'sourceDirectory': '/path/to/my_images'}]
+
+# Configure extensions in the master config for console customization
+# See: https://docs.openshift.com/enterprise/latest/install_config/web_console_customization.html#serving-static-files
+#openshift_master_oauth_template=/path/to/login-template.html
+
+# Configure metricsPublicURL in the master config for cluster metrics
+# See: https://docs.openshift.com/enterprise/latest/install_config/cluster_metrics.html
+#openshift_master_metrics_public_url=https://hawkular-metrics.example.com/hawkular/metrics
```

Comment:
We likely need to link this to https://trello.com/c/tZ0TYoJr/64-automated-metrics-installation-productization or provide a note that Ansible can install metrics for you (to which this setting is important).